extensions/elixir/doc_builder.exs

```diff
@@ -0,0 +1,255 @@
+# Script responsible for generating .ts files for each of Elixir's modules documentation.
```

Comment:
Not sure if this file should stay as part of the extension. It is used to generate the documentation files, for example, `src/docs/atom.ts`, but after those files are generated it is not needed for the extension to work. As such, I'm ok with moving it to a Github Gist or a different repository 👍